src/registrar/templates/domain_detail.html

```diff
@@ -20,7 +20,7 @@
   {% endblock breadcrumb %}
 
   {{ block.super }}
-  <div class="margin-top-4 tablet:grid-col-10">
+  <div class="margin-top-4 tablet:grid-col-10 maxw-fit-content">
```

Comment:
THIS is the override

Reply:
Actually, that needs to go.  You found one I forgot to strip out.  I had this as the override in alot of places, but it made it so the original sizing (before widescreen) was not preserved.  Design wants the max width for these to be *exactly* the same as before.  Hence the comparatively less straightforward approach to max-width.

Reply:
UPDATE: I took it out and then remembered why it is there...it corrects a different set of inconsistencies we have in how content for these pages calculate their max widths *within* the column.  Some content is shorter than the actual column width.  Others are not.  This corrects the misalignment.